library/src/tensor_operation_instance/gpu/pool3d_fwd/device_max_pool3d_fwd_ndhwc_f16_instance.cpp

```diff
@@ -25,7 +25,7 @@ void add_device_pool3d_fwd_ndhwc_index_f16_instances(
         instances)
 {
     add_device_operation_instances(
-        instances, device_pool3d_fwd_ndhwc_instances<F16, F16, I32, F16, ReduceOpId, true>{});
+        instances, device_pool3d_fwd_ndhwc_instances<F16, F16, I32, F32, ReduceOpId, true>{});
```

Comment:
Since Max operation does not involves any accumulation in fwd phase, there's no need for wider range compute data type.